ckanext/switzerland/helpers/plugin_utils.py

```diff
@@ -80,7 +80,7 @@ def ogdch_prepare_search_data_for_index(search_data):  # noqa
     search_data['res_format'] = ogdch_format_utils.prepare_formats_for_index(
         resources=validated_dict[u'resources']
     )
-    search_data['res_rights'] = [ogdch_term_utils.simplify_terms_of_use(r['rights']) for r in validated_dict[u'resources'] if 'rights' in r.keys()]  # noqa
+    search_data['res_license'] = [ogdch_term_utils.simplify_terms_of_use(r['license']) for r in validated_dict[u'resources'] if 'license' in r.keys()]  # noqa
```

Comment:
could we break this line up please? i know we use `# noqa` on most of the long lines in this function, but this line is 159 characters long and that's twice the default line length in python, it seems a bit much!

Reply:
sure :)

Reply:
done